biztown/src/components/TableRow.js

```diff
@@ -1,96 +1,89 @@
-import React from "react"
```

Comment:
naming nit: can we update this from TableRow to something more specific? Maybe SurveyTableRow? LikertTableRow?